docs/UserGuide.md

```diff
@@ -425,33 +426,44 @@ _The display reflects the updated log entries for the student with ID #000001._
 
 **Q**: What are the system requirements for TuteeTally?<br>
 **A**: TuteeTally requires Java 11 or above to run. It is compatible with Windows, MacOS, and Linux operating systems.
 
+**Q**: Can I back up my TuteeTally data?
+**A**: Yes, it is recommended to regularly back up your tuteetally.json data file by copying it to a secure location. This ensures that your data can be restored in case of software or hardware issues.
+
+**Q**: What are the common mistakes to avoid when using TuteeTally?
+**A**: Common mistakes include inputting commands in the wrong format,in such instances you may reference either the error message or this User Guide on the correct format.
+
+**Q**: How can I recover deleted student data?
+**A**: Once student data is deleted, it cannot be recovered through the application. Regular backups of your data file are recommended to prevent loss of data.
+
+**Q**: What happens if I encounter a bug?
+**A**: You may report any bugs on our [GitHub](https://github.com/AY2324S2-CS2103T-F10-2/tp/issues). If possible, do provide a detailed description of the problem, steps to reproduce it, and any screenshots.
 --------------------------------------------------------------------------------------------------------------------
 
 ## Known issues
 
 1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
-2. **When inputting long names above 220 characters**, the ID number of the students will be blocked even after the work around.[above](#adding-a-student-add)
-3. **When using the payment commands**, the display will round the payment amounts to the nearest $0.01. So if you enter a payment of $0.001, it will be displayed as $0.00. However, the application will accurately track the exact amounts entered, without rounding.
-4. **Empty white block when maximised**, currently there is an empty white block in the bottom when the app is maximised
-5. **Empty white block in log when viewing students details**, currently there is an empty white block when viewing a student with no logs.
+2. **When using the payment commands**, the display will round the payment amounts to the nearest $0.01. So if you enter a payment of $0.001, it will be displayed as $0.00. However, the application will accurately track the exact amounts entered, without rounding.
+3. **Empty white block when maximised**, currently there is an empty white block in the bottom when the app is maximised
+4. **Empty white block in log when viewing students details**, currently there is an empty white block when viewing a student with no logs.
+5.* *Summary Stats Window**, currently it shows the exact value. 
```

Comment:
Might want to elaborate what "exact value" means in the form of "shows the exact value instead of..."

if not it doesnt sound like an issue!

Reply:
Thank you, I've change it le